RELEASE_NOTES.md

```diff
@@ -1,5 +1,18 @@
 ﻿RELEASE NOTES
 
+**Version**: 1.3.19.1
+
+This version requires:
+* tk-unity v1.1
+* Shotgun for Unity 1.0 (com.unity.integrations.shotgun)
+* Python for Unity 4.0.0-exp.5
```

Comment:
This is misleading for users. Isn't the Python package automatically pulled by Unity since it is a dependency in the Shotgun `package.json`?

Please remove this line